data-pipeline/check_db.py

```
@@ -1,40 +1,40 @@
         print(f"Locations: {loc_count}")
         print(f"Readings: {reading_count}")
-        
+
         if reading_count > 0:
             print("SUCCESS: Data found in database.")
         else:
             print("FAILURE: No data found in readings table.")
-            
+
         conn.close()
     except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)